tutorials/appstudio-devspace-fiori-create/appstudio-devspace-fiori-create.md

```diff
@@ -64,7 +66,9 @@ Dev spaces are like isolated virtual machines in the cloud that can be instantly
 
 The SAP Fiori dev space opens and the **Welcome** tab appears.
 
-!![Open Dev Space](03-01-02 AppStudio Welcome Tab.jpg)
+!![Open Dev Space](AppStudio Welcome Tab_.jpg)
```

Comment:
Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

Reply:
done